### PR TITLE
[ExchangeConfig] Add cryptocurrencies attribute 

### PR DESCRIPTION
**Pull request by Herklos**
Requires #54

Related to https://github.com/Drakkar-Software/OctoBot-Evaluators/pull/13

### CODE REVIEW

File: octobot_trading/exchanges/data/exchange_config_data.py [hunk 1]
```diff
@@ -114,36 +114,36 @@ def _set_config_time_frame(self):
...
 
-    def _add_tradable_symbols_from_config(self, crypto_currency):
+    def _add_tradable_symbols_from_config(self, cryptocurrency):
         return [
             symbol
-            for symbol in self.config[CONFIG_CRYPTO_CURRENCIES][crypto_currency][CONFIG_CRYPTO_ADD]
+            for symbol in self.config[CONFIG_CRYPTO_CURRENCIES][cryptocurrency][CONFIG_CRYPTO_ADD]
             if self.exchange_manager.symbol_exists(symbol)
-               and symbol not in self.traded_crypto_currencies_pairs[crypto_currency]
+            and symbol not in self.traded_cryptocurrencies[cryptocurrency]
```
Comment (GuillaumeDSM) [inline octobot_trading/exchanges/data/exchange_config_data.py]: Looks like we lost a tab here :)

File: octobot_trading/exchanges/exchanges.py [hunk 2]
```diff
@@ -32,23 +32,13 @@ def __init__(self):
...
 
     def add_exchange(self, exchange_manager) -> None:
-        if exchange_manager.exchange.name not in self.exchanges:
-            self.exchanges[exchange_manager.exchange.name] = {}
+        self.exchanges[exchange_manager.exchange.name] = ExchangeConfiguration(exchange_manager)
 
-        self.exchanges[exchange_manager.exchange.name][exchange_manager.id] = ExchangeConfiguration(exchange_manager)
-
-    def get_exchange(self, exchange_name, exchange_manager_id) -> ExchangeConfiguration:
-        return self.exchanges[exchange_name][exchange_manager_id]
```
Comment (GuillaumeDSM) [inline octobot_trading/exchanges/exchanges.py]: Why removing exchange_manager_id ?

File: octobot_trading/exchanges/exchanges.py [hunk 3]
```diff
@@ -22,7 +22,7 @@ class ExchangeConfiguration:
     def __init__(self, exchange_manager):
         self.exchange_manager = exchange_manager
         self.exchange_name = exchange_manager.exchange.name
-        self.id = exchange_manager.id
```
Comment (GuillaumeDSM) [inline octobot_trading/exchanges/exchanges.py]: Why removing self.id ?

File: octobot_trading/exchanges/exchanges.py [hunk 4]
```diff
@@ -32,23 +32,13 @@ def __init__(self):
         self.exchanges = {}
 
     def add_exchange(self, exchange_manager) -> None:
-        if exchange_manager.exchange.name not in self.exchanges:
-            self.exchanges[exchange_manager.exchange.name] = {}
+        self.exchanges[exchange_manager.exchange.name] = ExchangeConfiguration(exchange_manager)
 
-        self.exchanges[exchange_manager.exchange.name][exchange_manager.id] = ExchangeConfiguration(exchange_manager)
```
Comment (GuillaumeDSM) [inline octobot_trading/exchanges/exchanges.py]: Why removing exchange_manager_id ?

File: octobot_trading/exchanges/exchanges.py [hunk 5]
```diff
@@ -32,23 +32,13 @@ def __init__(self):
...
-        return self.exchanges[exchange_name][exchange_manager_id]
-
-    def get_exchanges(self, exchange_name) -> dict:
+    def get_exchange(self, exchange_name) -> ExchangeConfiguration:
         return self.exchanges[exchange_name]
 
-    def del_exchange(self, exchange_name, exchange_manager_id) -> None:
+    def del_exchange(self, exchange_name) -> None:
         try:
-            self.exchanges[exchange_name].pop(exchange_manager_id, None)
```
Comment (GuillaumeDSM) [inline octobot_trading/exchanges/exchanges.py]: Why removing exchange_manager_id ?

File: octobot_trading/exchanges/exchanges.py [hunk 6]
```diff
@@ -32,23 +32,13 @@ def __init__(self):
...
-
-            if not self.exchanges[exchange_name]:
-                self.exchanges.pop(exchange_name, None)
+            self.exchanges.pop(exchange_name, None)
         except KeyError:
-            get_logger(self.__class__.__name__).warning(f"Can't del exchange {exchange_name} "
```
Comment (GuillaumeDSM) [inline octobot_trading/exchanges/exchanges.py]: Why removing exchange_manager_id ?

File: octobot_trading/exchanges/data/exchange_config_data.py (hunk 1 above)
Comment (Herklos) [reply octobot_trading/exchanges/data/exchange_config_data.py]: Yes there was a additionnal tab

File: octobot_trading/exchanges/exchanges.py (hunk 3 above)
Comment (Herklos) [reply octobot_trading/exchanges/exchanges.py]: Hum I don't understand... I missed something with git probably

Comment (Herklos) [reply octobot_trading/exchanges/exchanges.py]: Thanks, it's fixed now.

**Pull request merged**